app/src/utils/Utils.ts

```diff
@@ -337,6 +346,36 @@ export const pluralize = (quantity: number, word: string, singularSuffix = '', p
   return `${word}${quantity === 1 ? singularSuffix : pluralSuffix}`;
 };
 
+type nullableString = string | null | undefined;
```

Comment:
Nit: use Pascal case for types. I would argue that this type is unneeded anyway, because

1. The name doesn't accurately reflect what it is (I wouldn't expect a "nullable string" to be possibly `undefined` - rather I expect it to either be defined or to be `null`), and;
2. Reading `NullableString` is less clear than `string | null` (case in point, `NullableString` is hiding an assumption)

I would either explicitly use `string | null | undefined`, or something like

```javascript
type DateRange = [string, string | null | undefined]
```
which represents exactly two values, the latter of which can be falsy.

Reply:
I just did this to get rid of a code smell. Kind of an unhelpful sonar rule, since I agree with everything you've said. I'll just change it back and forget about the code smell.

Reply:
Yeah this code smell seems a but dubious. We can just mark it as "Won't Fix"